takahe/settings.py

```diff
@@ -404,14 +404,18 @@ class Config:
     parsed = urllib.parse.urlparse(SETUP.MEDIA_BACKEND)
     query = urllib.parse.parse_qs(parsed.query)
     if parsed.scheme == "gs":
-        DEFAULT_FILE_STORAGE = "core.uploads.TakaheGoogleCloudStorage"
+        STORAGES["default"] = {
+            "BACKEND": "core.uploads.TakaheGoogleCloudStorage",
+        }
```

Comment:
This way you don't overwrite other 'default' keys:
```suggestion
        STORAGES["default"]["BACKEND"] = "core.uploads.TakaheGoogleCloudStorage"
```
The same is valid for below changes.

Reply:
Thanks! Applied this